docs/source/release/v4.0.0/index.rst

```diff
@@ -26,18 +26,32 @@ Changes
 -------
```

Comment:
Higher up in the this contains a line `Here is a load of stuff we have done.` and does not have the usual introductory paragraph/picture the release notes normally have. I think this is probably out of scope for this PR but just wanted to check.

Reply:
I have a note on the project board to fill this in before the release is finalized. I think I'd prefer to do that in a separate issue if that's okay?

Reply:
That's fine by me just wanted to double check it didn't slip through the cracks